testing/debug/Kconfig

```diff
@@ -0,0 +1,32 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config TESTING_DEBUG
+	tristate "Debug test"
+	default n
+	depends on ARCH_HAVE_DEBUG
```

Comment:
remove, it's enough to depends on LIB_GDBSTUB

Reply:
No, it's possible that LIB_GDBSTUB enabled but ARCH_HAVE_DEBUG disabled.